Sources/Transmute/transmute().swift

```diff
@@ -31,7 +31,12 @@ public func transmute(packages: [Package]) throws -> ([Module], [Product]) {
                 //        depend upon 'Utility', and hope that no users define
                 //        test modules named 'Functional'.
                 testModule.dependencies = modules.filter{ $0.name == "Utility" }
-            } else {
+            }
+            else if testModule.basename == "Transmute" {
```

Comment:
nit-pick: For consistency's sake, it seems most other `else` statements are on the same line--that is:

```swift
if (...) {
    // ...
} else {
    // ...
}
```